src/compiler/commandLineParser.ts

```diff
@@ -332,6 +332,40 @@ namespace ts {
             name: "noImplicitUseStrict",
             type: "boolean",
             description: Diagnostics.Do_not_emit_use_strict_directives_in_module_output
+        },
+        {
+            name: "lib",
+            shortName: "l",
+            type: {
+                // JavaScript only
+                "es5": "lib.es5.d.ts",
+                "es6": "lib.es6.d.ts",

```

Comment:
thinking out loud here; to avoid the breaking change with lib.es6.d.ts, we can switch this to use the other one so `"es6" : "lib.es6.includes.d.ts"` and leave `lib.es6.full.d.ts` as `lib.es6.d.ts`